stubs/2.7/json.pyi

```diff
@@ -6,7 +6,7 @@ class JSONDecodeError(object):
     def loads(self, s: str) -> Any: ...
     def load(self, fp: IO[str]) -> Any: ...
 
-def dumps(obj: Any, sort_keys: bool = None, indent: int = None) -> str: ...
-def dump(obj: Any, fp: IO[str], *args: Any, **kwds: Any) -> None: ...
+def dumps(obj, skipkeys=False, ensure_ascii=True, check_circular=True, allow_nan=True, indent: int = None, separators: Tuple[str, str] = None, encoding='utf-8', sort_keys=False) -> str: ...
```

Comment:
Mypy doesn't infer argument types from default argument values, so it would make sense to annotate all the arguments.